File: waterproof/src/main/java/mx/yellowme/waterproof/mvp/simple/SimplePresenter.java
package mx.yellowme.waterproof.mvp.simple;

import mx.yellowme.waterproof.mvp.BasePresenter;

public abstract class SimplePresenter<Model> extends BasePresenter<SimpleView<Model>> {

    public SimplePresenter(SimpleView<Model> mView) {
        super(mView);
    }

    public abstract void loadElement(boolean forceUpdate);
}

File: waterproof/src/main/java/mx/yellowme/waterproof/mvp/simple/SimplePresenter.kt
package mx.yellowme.waterproof.mvp.simple

import mx.yellowme.waterproof.mvp.WpPresenter

abstract class SimplePresenter<Model>(mView: SimpleView<Model>) : WpPresenter<SimpleView<Model>>(mView) {

    abstract fun loadElement(forceUpdate: Boolean)
}
